experimenter/databases/d3m_mtl.py

```diff
@@ -23,17 +23,17 @@ def __init__(self) -> None:
         self._post_url = D3M_MTL_DB_POST_URL
         # This env var allows code calling this class to be run during
         # unit tests without actually saving to the production DB.
-        self.should_save = config.SAVE_TO_D3M
+        self.should_save = config.D3MConfig().save_to_d3m
```

Comment:
This will raise an error. There is no `config.D3MConfig()`. Could you add a test to make sure that this `__init__` method runs without failing?